src/python/module/smartredis/client.py

```diff
@@ -466,7 +466,7 @@ def get_script(self, name: str) -> str:
 
     @exception_handler
     def run_script(
-        self, name: str, fn_name: str, inputs: t.List[str], outputs: t.List[str]
+        self, name: str, fn_name: str, inputs: str, outputs: str
```

Comment:
Lists are still permitted, so this typehint is not correct. You need the compound form that allows both strings and lists of strings